logicrunner/builtin/contract/migrationadmin/migrationadmin.go

```diff
@@ -64,21 +70,66 @@ func (mA *MigrationAdmin) MigrationAdminCall(params map[string]interface{}, name
 	return nil, fmt.Errorf("unknown method: migration.'%s'", nameMethod)
 }
 
-func (mA *MigrationAdmin) activateDaemonCall(params map[string]interface{}, memberRef insolar.Reference) (interface{}, error) {
-	migrationDaemon, ok := params["reference"].(string)
-	if !ok && len(migrationDaemon) == 0 {
+func (mA *MigrationAdmin) changeStatusCall(params map[string]interface{}, caller insolar.Reference) (*migrationdaemon.MigrationDaemon, error) {
```

Comment:
Why `changeStatusCall` ? its not changing anything as I can see

Reply:
rename